WebRoot/public/login_history.jsp

```diff
@@ -0,0 +1,105 @@
+<style>
+body {
+	margin-top: 10%;
+}
+hr {
+	width: 50%;
+}
+.divTable{
+	display: table;
+	width: 50%;
+}
+.divTableRow {
+	background-color: #F2F2F2;
+	display: table-row;
+}
+.divTableHeading {
+	background-color: #EEE;
+	display: table-header-group;
+}
+.divTableCell, .divTableHead {
+	border: 1px solid #999999;
+	display: table-cell;
+	padding: 3px 10px;
+}
+.divTableHeading {
+	background-color: #EEE;
+	display: table-header-group;
+	font-weight: bold;
+}
+.divTableFoot {
+	background-color: #EEE;
+	display: table-footer-group;
+	font-weight: bold;
+}
+.divTableBody {
+	display: table-row-group;
+}
+</style>
+<div align="center">
+
```

Comment:
When the page is called via jsp:forward, the jsp page should have html, (head,) and body tags.